<ui version="4.0" >
 <class>SynonymWidget</class>
 <widget class="QWidget" name="SynonymWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>196</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QLabel" name="synonymLabel" >
     <property name="text" >
      <string>Synonyms</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="listView" />
   </item>
   <item>
    <widget class="QPushButton" name="synonymButton" >
     <property name="text" >
      <string>Synonym</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
